.github/workflows/check-plugin.yml

```diff
@@ -26,12 +26,19 @@ jobs:
           echo ::set-env name=CHIME_MESSAGE::$(cat /tmp/chime_message.md)
           exit `cat /tmp/plugin_status.check`
 
-      - name: Send Chime Message
+#      - name: Send Chime Message
+#        if: ${{ always() }}
+#        uses: ros-tooling/action-amazon-chime@master
+#        with:
+#          message: ${{ env.CHIME_MESSAGE }}
+#          webhook: ${{ secrets.CHIME_ODFE_RELEASE }}
```

Comment:
remove these if they won't be used anymore

Reply:
We still have chime do we really want to completely remove it?

Reply:
I don't see the need to keep it. Let's remove it.

Reply:
Resolved.